packages/file_selector/file_selector_linux/lib/file_selector_linux.dart

```diff
@@ -82,19 +82,37 @@ class FileSelectorLinux extends FileSelectorPlatform {
     String? initialDirectory,
     String? suggestedName,
     String? confirmButtonText,
+  }) async {
+    return (await getSaveLocation(
```

Comment:
Maybe split the await and return on this one?

```dart
final FileSaveLocation? saveLocation = await getSaveLocation...
return saveLocation?.path;
```

(There's a similar one in `packages/file_selector/file_selector_macos/lib/file_selector_macos.dart`, `packages/file_selector/file_selector_windows/lib/file_selector_windows.dart`)

Reply:
Good call, fixed in all three places.